test/email-alert.test.js

```diff
@@ -0,0 +1,25 @@
+const emailAlerts = require('./../src/email-alert');
+
+const {expect} = require('chai');
+
+it('Send mail on TOO_LOW alert ', () => {
+  expect(emailAlerts.sendToEmail('TOO_LOW'))
+      .equals('TOO_LOW_STATUS_EMAIL_ALERT_SENT');
+});
+
+it('Send mail on TOO_HIGH alert ', () => {
+  expect(emailAlerts.sendToEmail('TOO_HIGH'))

```

Comment:
This would still leave the email contents to be tested by sending an actual email and checking the inbox.
E.g., if someone has copy-pasted the message for both high and low, then this test will still pass, while the email content will be wrong.
What would it take to test the email content? How about separating the conversion logic from the `TOO_HIGH` to human-readable form? That can be proven without actually having to send the email.